userspace/libsinsp/parsers.cpp

```diff
@@ -2384,7 +2384,7 @@ std::string sinsp_parser::parse_dirfd(sinsp_evt *evt, std::string_view name, int
 		return "<UNKNOWN>";
 	}
 
-	if(fdinfo->m_name.back() == '/') {
+	if(!fdinfo->m_name.empty() && fdinfo->m_name.back() == '/') {
```

Comment:
In both fixes, if `m_name` is empty i return `"/"`; perhaps it is better to just return the empty string?

Reply:
Decided together with @jasondellaluce to just return an empty string in these cases.